RESOURCES/FEATURE_FLAGS.md

```diff
@@ -57,9 +62,26 @@ These features flags are **safe for production** and have been tested.
 - SQL_VALIDATORS_BY_ENGINE [(docs)](https://superset.apache.org/docs/installation/sql-templating)
 - SQLLAB_BACKEND_PERSISTENCE
 - THUMBNAILS [(docs)](https://superset.apache.org/docs/installation/cache)
+- ENABLE_EXPLORE_DRAG_AND_DROP
+- ENABLE_DND_WITH_CLICK_UX
+- EMBEDDABLE_CHARTS
 
 ## Deprecated Flags
 These features flags currently default to True and **will be removed in a future major release**. For this current release you can turn them off by setting your config to False, but it is advised to remove or set these flags in your local configuration to **True** so that you do not experience any unexpected changes in a future release.
 
 - ALLOW_DASHBOARD_DOMAIN_SHARDING
 - DISPLAY_MARKDOWN_HTML
+- FORCE_DATABASE_CONNECTIONS_SSL
+
+## In need of categorization
+- DRUID_JOINS
+- DISABLE_LEGACY_DATASOURCE_EDITOR
+- DASHBOARD_FILTERS_EXPERIMENTAL
+- EMBEDDED_SUPERSET
+- ENABLE_ADVANCED_DATA_TYPES
+- ALERTS_ATTACH_REPORTS
+- ENFORCE_DB_ENCRYPTION_UI
+- ALLOW_FULL_CSV_EXPORT
+- ALLOW_ADHOC_SUBQUERY
+- RLS_IN_SQLLAB
+- CACHE_IMPERSONATION
```

Comment:
Hi @rusackas, I would put `CACHE_IMPERSONATION` in the **In Testing** category.

Reply:
`DRUID_JOINS` should be stable and default to true. Only people using older versions of Druid should turn it off.

`RLS_IN_SQLLAB` should be "in testing", it allows RLS rules to be applied to arbitrary queries in SQL Lab, but since it requires parsing the SQL it's more-or-less experimental.

Reply:
Thanks both of you! These changes have been made. More changes still to come!